packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -93,6 +93,7 @@ export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}> impl
         className={className}
         ref={this._root}
         aria-labelledby={ariaLabelledBy}
+        aria-modal="true"
```

Comment:
>aria-modal="true" [](start = 8, length = 17)

aria-modal only applies to window roles (alertdialog and dialog), see: [https://www.w3.org/TR/wai-aria-1.1/#aria-modal](https://www.w3.org/TR/wai-aria-1.1/#aria-modal) (the "used in roles" section). that being said, there's divProps on the FocusTrapZone so a user could already pass aria-modal in

Reply:
The applicable text from that link says:
> The aria-modal attribute is used to indicate that the presence of a "modal" element precludes usage of other content on the page. For example, when a modal dialog is displayed, it is expected that the user's interaction is limited to the contents of the dialog, until the modal dialog loses focus or is no longer displayed.

They use modal in quotes and refer to modal-like elements rather than exactly and specifically the `modal` role. In this case, FocusTrapZone (and other higher order elements such as Panel) are in fact "modal" elements that are precluding usage of other content on the page. The linked issue is also saying this, as well, and I agree with the reasoning.

Reply:
Hmm, I don't disagree with the general reasoning there, but just because TAB is bound to the FTZ, it doesn't mean that it is modal. The ribbon in office, for example, loops focus, but it isn't modal and the user can click/touch or use CTRL + F6 to navigate out of that content. Modal usually mean that the rest of the application can't be interacted with, with isn't always the case with FTZ

Reply:
Good points as well, I had thought of tying the `aria-modal` attribute to some FTZ props as well, but I'm not sure if that would be reliable.

What should we expect scan mode to do in the ribbon case? Should a user be able to navigate outside of the ribbon in scan mode?

Reply:
I would expect it to go outside of the ribbon (you could imagine a user who uses scan mode to navigate around the app and doesn't use CTRL +f6 (in the office online case) to navigate around the app, then they are stuck in the ribbon if an AT doesn't move outside of the modal area with virtual navigation turned on... (which would be really bad)

Reply:
Related to my other comment (and yours), I'm thinking FocusTrapZone can't possibly know when to apply `aria-modal` accurately. I think I will close this PR and resolve the issue as being the responsibility of the FocusTrapZone consumer.